src/main/kotlin/no/nav/familie/ba/sak/kjerne/beregning/TilkjentYtelseUtils.kt

```diff
@@ -172,7 +175,15 @@ object TilkjentYtelseUtils {
                             }
                 }
 
-        tilkjentYtelse.andelerTilkjentYtelse.addAll(andelerTilkjentYtelse)
+        val utvidetBarnetrygdAndeler = UtvidetBarnetrygdGenerator(behandlingId = vilkårsvurdering.behandling.id,
```

Comment:
```suggestion
        val andelerTilkjentYtelseSøker = UtvidetBarnetrygdGenerator(behandlingId = vilkårsvurdering.behandling.id,
```
Er det greit at det heter det samme 🤷‍♂️